LayerNorm Registration Example

bwasti opened the pull request:
Summary: This diff includes an example registration of a caffe2 op in torch.  A previous attempt ran into a static initialization order bug.

Differential Revision: D13854304

smessmer reviewed (approved):
Looks good. Please make sure any remaining CI errors are unrelated before landing.

bwasti committed:
Enable undefined at::Tensor to be passed as Output

Differential Revision: D13946950

fbshipit-source-id: ae3a6d1e4ae404f3432627540eaac0390583d355

bwasti committed:
LayerNorm Registration Example (#16478)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/16478

This diff includes an example registration of a caffe2 op in torch.  A previous attempt ran into a static initialization order bug.

Reviewed By: smessmer

Differential Revision: D13854304

fbshipit-source-id: 5b8b053e6f1207bb859e537f71a22597422fbc26

ezyang commented:
This PR broke ASAN and was reverted
